app/models/account/credit_card.rb

```diff
@@ -134,18 +131,26 @@ def notify_credit_card_change
 
   private
 
+  delegate :payment_gateway_setting_exists?, to: :provider_account, prefix: true, allow_nil: true
```

Comment:
Oops. I see the delegation here now. Please ignore https://github.com/3scale/porta/pull/798#pullrequestreview-259603670.